gratipay/models/team/__init__.py

```diff
@@ -167,31 +167,31 @@ def get_payment_distribution(self):
 
 
     def update(self, **kw):
-      updateable = frozenset(['name', 'product_or_service', 'homepage',
-                              'onboarding_url'])
-
-      cols, vals = zip(*kw.items())
-      assert set(cols).issubset(updateable)
-
-      old_value = {}
-      for col in cols:
-        old_value[col] = getattr(self, col)
-
-      cols = ', '.join(cols)
-      placeholders = ', '.join(['%s']*len(vals))
-
-      with self.db.get_cursor() as c:
-        c.run("""
-          UPDATE teams
-             SET ({0}) = ({1})
-           WHERE id = %s
-          """.format(cols, placeholders), vals+(self.id,)
-        )
-        add_event(c, 'team', dict( action='update'
-                                 , id=self.id
-                                 , **old_value
-                                  ))
-        self.set_attributes(**kw)
+        updateable = frozenset(['name', 'product_or_service', 'homepage',
+                                'onboarding_url'])
+
+        cols, vals = zip(*kw.items())
+        assert set(cols).issubset(updateable)
+
+        old_value = {}
+        for col in cols:
+            old_value[col] = getattr(self, col)
+
+        cols = ', '.join(cols)
+        placeholders = ', '.join(['%s']*len(vals))
+
+        with self.db.get_cursor() as c:
+            c.run("""
+              UPDATE teams
+                 SET ({0}) = ({1})
+               WHERE id = %s
+              """.format(cols, placeholders), vals+(self.id,)
+            )
+            add_event(c, 'team', dict( action='update'
+                                     , id=self.id
+                                     , **old_value
+                                      ))
+            self.set_attributes(**kw)
 
```

Comment:
Yes. When I started this PR I thought I was going to be changing this file, so I made f0b74e10c0c25febfd24a6e7a7bd16f025bd52f2 in order to standardize the whitespace first. I didn't end up changing the code but I left the formatting change in place.